packages/cms-cli/commands/customObject/schema/fetch-all.js

```diff
@@ -0,0 +1,56 @@
+const {
+  loadConfig,
+  validateConfig,
+  checkAndWarnGitInclusion,
+} = require('@hubspot/cms-lib');
+const { logger } = require('@hubspot/cms-lib/logger');
+const { logErrorInstance } = require('@hubspot/cms-lib/errorHandlers');
+
+const { validatePortal } = require('../../../lib/validation');
+const { trackCommandUsage } = require('../../../lib/usageTracking');
+const { setLogLevel, getPortalId } = require('../../../lib/commonOpts');
+const { logDebugInfo } = require('../../../lib/debugInfo');
+const { downloadSchemas } = require('@hubspot/cms-lib/schema');
+
+exports.command = 'fetch-all [dest]';
+exports.describe = 'Fetch all Custom Object Schema for a portal';
+
+exports.handler = async options => {
+  setLogLevel(options);
+  logDebugInfo(options);
+  loadConfig(options.config);
+  checkAndWarnGitInclusion();
+
+  if (!(validateConfig() && (await validatePortal(options)))) {
+    process.exit(1);
+  }
+  const portalId = getPortalId(options);
+
+  trackCommandUsage('schema-fetch-all', null, portalId);
+
+  try {
+    await downloadSchemas(portalId, options.dest);
+  } catch (e) {
+    logErrorInstance(e);
+    logger.error('Unable to fetch schemas');
+  }
+};
+
+exports.builder = yargs => {
+  yargs.example([
+    [
+      '$0 custom-object schema fetch-all',
+      'Fetch all schemas for a portal and put them in the current working directory',
+    ],
+    [
+      '$0 custom-object schema fetch-all my/folder',
+      'Fetch all schemas for a portal and put them in a directory named my/folder',
+    ],
+  ]);
+
+  yargs.positional('dest', {
+    describe:
+      'Local destination folder to write schemas to.  If omitted, current working directory will be used',
```

Comment:
Maybe "Local folder where schemas will be written." to avoid ending the sentence with a preposition?